api/ssh.go

```diff
@@ -0,0 +1,53 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type SSH struct {
```

Comment:
All of the types and functions are missing their comment string for documentation